### ISSUE TITLE
CV package no longer supported. Close repo please

### ISSUE THREAD

**Issue by j-martens** [CatalystCode/Handwriting#4]
this package is no longer part of Azure ML. Please close repo to minimize confusion and search issues related to deprecated functionality. Thank you!!

**Comment by ChiragSoni95** [comment]
@j-martens Could you tell, where can we find CVTK package now if I Want to run this notebook?
I get that the step 2 where MSFT Cognitive Service CV API will be used to transcribe actual text inside each box of text detected in step 1 using CVTK. How can I accomplish step 1 if CVTK package is no longer part of Azure ML?

**Comment by j-martens** [comment]
@neta2504 can you provide any details here?

**Comment by ChiragSoni95** [comment]
Yes @neta2504, if you could tell us the alternative way to retrieve the bounding box co-ordinates of each object using the trained Faster R-CNN model that would be of a great help.
**P.S**: One way could be using CNTK and train Faster R-CNN on our custom dataset, but I am unaware of the functionality to retrieve the Bounding box co-ordinates for ROI's of each test image as that was shown using CVTK package in this notebook.

**Comment by niloofers** [comment]
> Yes @neta2504, if you could tell us the alternative way to retrieve the bounding box co-ordinates of each object using the trained Faster R-CNN model that would be of a great help.
> **P.S**: One way could be using CNTK and train Faster R-CNN on our custom dataset, but I am unaware of the functionality to retrieve the Bounding box co-ordinates for ROI's of each test image as that was shown using CVTK package in this notebook.

ChiragSoni95, did you find an alternative for the cvtk package? Even I get the the module not found error when trying to run the program.
